frontend/screens/CreateAccount.js

```diff
@@ -286,12 +290,12 @@ const styles = StyleSheet.create({
   mediumText: {
     fontFamily: 'CircularStd-Medium',
```

Comment:
some fonts are in global style sheet, use those if not put like boldFont in global stylesheet and use from there...if we want to ever change font later we just need to change global style sheet